Orchid.Jint/JsCallback.cs

```diff
@@ -69,8 +89,23 @@ public object Invoke(params object[] args)
                 jsArgs[i] = JsValue.FromObject(_context.Engine, args[i]);
             }
 
-            var result = _callback(jsThis, jsArgs);
-            return result.ToObject();
+            try
```

Comment:
seems like at least the callback invocation could be abstracted into a shared method.

Reply:
Ugh, yes. There is.... _a lot_ of this throughout jint. I want to clean that up as part of the ticket for allowing collections to be sent through callback invokes and the mess that forced spreading introduces :/

Reply:
👍